CanonicalLayouts/list-detail-compose/app/build.gradle

```diff
@@ -62,21 +62,22 @@ android {
 }
 
 dependencies {
-    def composeBom = platform('androidx.compose:compose-bom:2024.03.00')
+    def composeBom = platform('androidx.compose:compose-bom:2024.08.00')
     implementation(composeBom)
 
     implementation "com.google.accompanist:accompanist-adaptive:0.32.0"
-    implementation 'androidx.core:core-ktx:1.12.0'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.7.0'
-    implementation 'androidx.activity:activity-compose:1.8.2'
-    implementation "androidx.compose.foundation:foundation:1.6.4"
-    implementation "androidx.compose.ui:ui:1.6.4"
+    implementation 'androidx.core:core-ktx:1.13.1'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.8.4'
+    implementation 'androidx.activity:activity-compose:1.9.1'
+    implementation "androidx.compose.foundation:foundation:1.6.8"
+    implementation "androidx.compose.ui:ui:1.6.8"
     implementation "androidx.compose.ui:ui-tooling-preview"
-    implementation "androidx.window:window:1.2.0"
-    implementation 'androidx.compose.material3:material3:1.3.0-alpha03'
-    implementation 'androidx.compose.material3.adaptive:adaptive:1.0.0-alpha09'
-    implementation 'androidx.compose.material3.adaptive:adaptive-layout:1.0.0-alpha09'
-    implementation 'androidx.compose.material3.adaptive:adaptive-navigation:1.0.0-alpha09'
-    implementation "androidx.compose.material3:material3-window-size-class:1.3.0-alpha03"
+    implementation "androidx.window:window:1.3.0"
+    implementation 'androidx.compose.material3:material3:1.3.0-rc01'
+    implementation 'androidx.compose.material3.adaptive:adaptive:1.1.0-alpha01'
+    implementation 'androidx.compose.material3.adaptive:adaptive-layout:1.1.0-alpha01'
+    implementation 'androidx.compose.material3.adaptive:adaptive-navigation:1.1.0-alpha01'
+    implementation "androidx.compose.material3:material3-window-size-class:1.3.0-rc01"
```

Comment:
Is this library used anywhere now, with the `material3.adaptive:adaptive` library? If not, we should be able to just remove it as a dependency.

Reply:
Yes, it imports the Surface, I'd leave it on for the moment, WDYT?